src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1540,9 +1542,14 @@ public void pageRequestNoticeIsSentWhenPolicyDefinesPageRequestNoticeConfigurati
       .atMost(1, TimeUnit.SECONDS)
       .until(patronNoticesClient::getAll, Matchers.hasSize(1));
     List<JsonObject> sentNotices = patronNoticesClient.getAll();
+
+    List<String> expectedContextPaths =
+      new ArrayList<>(NoticeTokens.EXPECTED_USER_TOKENS);
+    expectedContextPaths.addAll(NoticeTokens.EXPECTED_ITEM_TOKENS);
+    expectedContextPaths.addAll(NoticeTokens.EXPECTED_REQUEST_TOKENS);
     MatcherAssert.assertThat(sentNotices,
       hasItems(
-        equalsToEmailPatronNotice(requester.getId(), pageConfirmationTemplateId)));
+        equalsToEmailPatronNotice(requester.getId(), pageConfirmationTemplateId, expectedContextPaths)));
```

Comment:
This appears to check that the notice contains the expected contextual properties. Do any of the tests check that the values of those properties are what is expected?

Reply:
There are couple of tests where the due date property is checked. But most of the tests just check for not null.

Reply:
How do we know that the notice is being sent with the correct context?